setupinfo.py

```diff
@@ -285,6 +285,7 @@ def print_libxml_error():
     print('Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?')
     if sys.platform in ('darwin',):
         print('Perhaps try: xcode-select --install')
+    print('Otherwise, is your C compiler installed and configured correctly?')
```

Comment:
Seems helpful, thanks. I'd move it up, though, before the macOS suggestion. (And I just noticed that the named C function is not the one we check for.)
```suggestion
    print("Could not find function xmlXPathInit in library libxml2. Is libxml2 installed?")
    print("Is your C compiler installed and configured correctly?")
    )
    if sys.platform in ('darwin',):
        print('Perhaps try: xcode-select --install')
```

Reply:
Makes sense! Glad to help.